conduit/pipeline.go

```diff
@@ -131,13 +131,33 @@ type pipelineImpl struct {
 	importer   *importers.Importer
 	processors []*processors.Processor
 	exporter   *exporters.Exporter
+	round      basics.Round
 }
 
 func (p *pipelineImpl) Start() error {
 	p.logger.Infof("Starting Pipeline Initialization")
 
 	// TODO Need to change interfaces to accept config of map[string]interface{}
 
+	exporterLogger := log.New()
+	exporterLogger.SetFormatter(
+		PluginLogFormatter{
+			Formatter: &log.JSONFormatter{
+				DisableHTMLEscape: true,
+			},
+			Type: "Exporter",
+			Name: (*p.exporter).Metadata().Name(),
+		},
+	)
+
+	jsonEncode := string(json.Encode(p.cfg.Exporter.Config))
+	err := (*p.exporter).Init(plugins.PluginConfig(jsonEncode), exporterLogger)
```

Comment:
Could you put a comment somewhere that this is imported first so that we can fetch the round?
Would we ever want multiple exporters, which round do we choose in that scenario?

Reply:
I'll update w/ comment.

I think it's hard to enumerate all of possible combinations of multiple exporters. I'd prefer to make it really easy to run a pipeline so that people can just spin up multiple pipelines if they need to run multiple exporters.